### PR TITLE
Added the 3 first Translation Providers

### PR DESCRIPTION
**Pull request by welcoMattic**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | 

This PR adds recipes for the new Translation Providers, added in 5.3 (https://github.com/symfony/symfony/pull/38475)

cc @fabpot @nicolas-grekas @Nyholm

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. Have a look at the Mailer packages. 

Basically, We have some config for `symfony/mailer`: 

```yaml
framework:
    mailer:
        dsn: '%env(MAILER_DSN)%'
```

And for a bridge (`symfony/mailjet`) they just add the `MAILER_DSN` environment variable.

File: symfony/translation/5.3/config/packages/translation.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+framework:
+    default_locale: en
+    translator:
+        default_path: '%kernel.project_dir%/translations'
+        fallbacks:
+            - en
+#        providers:
```
Comment (fabpot) [inline symfony/translation/5.3/config/packages/translation.yaml:7]: I think most people will only ever use 1 provider, so having something similar to what we have in Mailer as mentioned by Tobia sounds better.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: symfony/translation/5.3/config/packages/translation.yaml (hunk 1 above)
Comment (welcoMattic) [reply symfony/translation/5.3/config/packages/translation.yaml:7]: Did you mean only 1 exemple provider, like:
```yaml
        providers:
            crowdin:
                dsn: '%env(CROWDIN_DSN)%'
```

If yes, which one do we want to make an example of?

Comment (fabpot) [reply symfony/translation/5.3/config/packages/translation.yaml:7]: More something along the lines of:

```yaml
        provider: '%env(TRANSLATION_PROVIDER_DSN)%'
```

Comment (welcoMattic) [reply symfony/translation/5.3/config/packages/translation.yaml:7]: Yes, you can add many providers, for example to manage `admin` domain in Loco, and `messages` domain in Crowdin. That's why I added an example for each. But, I'm ok to put only one example under `providers:`

Comment (OskarStark) [reply symfony/translation/5.3/config/packages/translation.yaml:7]: I would add 2 but not all, like we do in notifier, but @fabpot please decide, thanks

Comment (welcoMattic) [reply symfony/translation/5.3/config/packages/translation.yaml:7]: I just commit the removal of 2 providers in the example. Is it ok for you to keep 2 providers here?

**Comment by welcoMattic** [comment]
Following up to https://github.com/symfony/symfony/issues/41374, I removed the PoEditor recipe

**Comment by welcoMattic** [comment]
@Nyholm @OskarStark As 5.3 final has been released today, we need to merge

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Review by fabpot — APPROVED** [review]

**Review by fabpot — COMMENTED** [review]
merge now

**Pull request merged**